digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/TrafficSignService.scala

```diff
@@ -300,11 +119,7 @@ class TrafficSignService(val roadLinkService: RoadLinkService, val userProvider:
       case old if old.bearing != updatedAsset.bearing || (old.lat != updatedAsset.lat || old.lon != updatedAsset.lon) || old.validityDirection != updatedAsset.validityDirection =>
         expireWithoutTransaction(id)
         val newId = OracleTrafficSignDao.create(setAssetPosition(updatedAsset, roadLink.geometry, value), value, username, roadLink.municipalityCode, vvhTimeStamp.getOrElse(VVHClient.createVVHTimeStamp()), roadLink.linkSource, old.createdBy, old.createdAt)
-        if(belongsToTurnRestriction(updatedAsset)) {
-          println(s"Creating manoeuvre on linkId: ${roadLink.linkId} from import traffic sign with id $newId" )
-          eventBus.publish("manoeuvre:expire", id)
-          eventBus.publish("manoeuvre:create", ManoeuvreProvider(getPersistedAssetsByIdsWithoutTransaction(Set(newId)).head, roadLink))
-        }
+        eventBus.publish("trafficSign:update", ((newId, roadLink), id))
```

Comment:
this  publish doesn't exist, should be replace by create and delete or create a update process